src/NServiceBus.Core/Pipeline/SatelliteDefinition.cs

```diff
@@ -16,17 +16,17 @@ public SatelliteDefinition(string name, string receiveAddress, TransportTransact
             RecoverabilityPolicy = recoverabilityPolicy;
             OnMessage = onMessage;
         }
+        
+        public string Name { get; set; }
 
-        public string Name { get; }
-
-        public string ReceiveAddress { get; }
+        public string ReceiveAddress { get; set; }
```

Comment:
Made them get set to be able to late bind certain settings to not pollute the feature configuration context methods with unnecessary knowledge